internal/provider/data_source_workspace_test.go

```diff
@@ -204,6 +205,13 @@ func TestAccTFEWorkspaceDataSource_readProjectIDNonDefault(t *testing.T) {
 }
 
 func testAccTFEWorkspaceDataSourceConfig(rInt int) string {
```

Comment:
I think its OK to specify `auto_apply_run_trigger` even if the API doesn't support it. Many enterprise installations will not.

Reply:
I think it's cool that this feature was enabled in the test environment